files/image_config/logrotate/logrotateOverride.conf

```diff
@@ -1,2 +1,4 @@
 [Unit]
-Requires=logrotate-config.service
\ No newline at end of file
+Requires=logrotate-config.service
+After=rsyslog.service
+Requires=rsyslog.service
```

Comment:
Instead of adding a dependency on rsyslog here, can you instead check to see if rsyslog is running before trying to send the SIGHUP signal, so that there's no error message? That way, this can run completely independently of rsyslog running or not.

(More so, orchagent directly writes to its own log files, and doesn't go through rsyslog. If rsyslog isn't running, we want those files to still get rotated.)

Reply:
@saiarcot895 I added a check on the pid file